linode_metadata/metadata_client.py

```diff
@@ -132,7 +132,8 @@ def _api_call(
 
             if self._token is None:
                 raise RuntimeError(
-                    "No token provided. Please use MetadataClient.refresh_token() to create new token."
+                    "No token provided. Please use "
+                    "MetadataClient.refresh_token() to create new token."
```

Comment:
This wasn't introduced in this PR but we can go ahead and fix it here 🙂 
```suggestion
                    "MetadataClient.refresh_token() to create a new token."
```

Reply:
Sounds good to me. Thanks for fixing it!